### ISSUE TITLE
[WAI] existing WAI content

### ISSUE THREAD

**Issue by shawna-slh** [w3c/accessibility-intro#4]
Please read through the existing/old WAI [Introduction to Web Accessibility](https://www.w3.org/WAI/intro/accessibility) carefully.

<s>
**What important points, phrases, ideas, etc. are on that page that are not in the [draft W3C page](https://w3c.github.io/accessibility-intro/)?**
Are they already covered elsewhere on the WAI website? _(If you don't know, that's fine :)_
Should they be included in the new WAI Intro page, or on other WAI pages?
</s>

Update:
* What important points, phrases, ideas, etc. do we want to make sure to keep when update and revised this page?
* What things can be cut or tersified?

**Comment by vmmiller** [comment]
I've already commented in the comparison of both pages section that this resource is really excellent.  I'd keep it as it addresses a more general audience and explains very clearly what web accessibility is about.

**Comment by Sylduch-Conseil** [comment]
Hello,
What I like on the [W3C draft page](https://w3c.github.io/accessibility-intro/) is the reference to the UN convention and the right for a person with disability to have access to information, like others. 
But then, the doc implies many other users, low bandwidth, search engines, who benefit from accessibility but for whom accessibility is not essential. When a site is not accessible, a person with disability has no access to the information. A search engine does. 
I heard the argument of SEO would not be an argument anymore. 
While reading the [Wai old accessibility page](https://www.w3.org/WAI/intro/accessibility), I see stronger arguments for people with disabilities that are not in the W3C page. 

> It is essential that the Web be accessible in order to provide equal access and equal opportunity to people with disabilities. An accessible Web can also help people with disabilities more actively participate in society.
The Web offers the possibility of unprecedented access to information and interaction for many people with disabilities. That is, the accessibility barriers to print, audio, and visual media can be much more easily overcome through Web technologies. 

I think these other benefits should be added at the end but not mixed with the main people concerned by web accessibility, that is people with disabilities and older people. 
Another important point of the old page is: 
> Another important consideration for organizations is that Web accessibility is required by laws and policies in some cases. WAI Web Accessibility Policy Resources links to resources for addressing legal and policy factors within organizations, including a list of relevant laws and policies around the world.

**Comment by notabene** [comment]
+1 for @Sylvie42 's suggestion to keep the very strong arguments ('It is essential [etc]') + law requirements.

Considering many companies now aim at worldwide deployment, legal requirements will soon become (fingers crossed) a de facto automatic requirement.
